Sources/Logging/Logging.swift

```diff
@@ -38,21 +38,61 @@ import WASILibc
 /// ```swift
 /// logger.info("Hello World!")
 /// ```
+
 public struct Logger {
+    /// Storage class to hold the label and log handler
+    @usableFromInline
+    internal final class Storage {
+        @usableFromInline
+        var label: String
+        @usableFromInline
+        var handler: LogHandler
+        @usableFromInline
+        init(label: String, handler: LogHandler) {
+            self.label = label
+            self.handler = handler
+        }
+    }
+    
+    @usableFromInline
+    internal var _storage: Storage
     @usableFromInline
-    var handler: LogHandler
+    internal var label: String {
```

Comment:
I think this needs to be `public` like before
```suggestion
    public var label: String {
```

Reply:
```'@usableFromInline' attribute can only be applied to internal or package declarations, but 'label' is public```, getting this error. 
Should I remove ```@usableFromInline``` or keep it ```internal```?

Reply:
Yeah remove `@usableFromInline` and keep it public